### ISSUE TITLE
Tutorial step_3 test failure

### ISSUE THREAD

**Issue by bobmyhill** [geodynamics/burnman#215]
***  End of file "/home/bob/projects/burnman/misc/ref/step_3.py.out" reached while trying to read line 1001.
***  File "step_3.py.tmp" has more lines than file "/home/bob/projects/burnman/misc/ref/step_3.py.out",
***  line 1001 is the last one read from file "step_3.py.tmp"

!  step_3.py ... FAIL
Check: /home/bob/projects/burnman/tutorial/step_3.py.tmp /home/bob/projects/burnman/misc/ref/step_3.py.out
----------------
##1001    <== /usr/lib/python2.7/dist-packages/matplotlib/axes/_base.py: UnicodeWarning: Unicode equal comparison failed to convert both arguments to Unicode - interpreting them as being unequal
          ==>

***  End of file "/home/bob/projects/burnman/misc/ref/step_3.py.out" reached while trying to read line 1001.
***  File "step_3.py.tmp" has more lines than file "/home/bob/projects/burnman/misc/ref/step_3.py.out",
***  line 1001 is the last one read from file "step_3.py.tmp"

+++  File "step_3.py.tmp" differs from file "/home/bob/projects/burnman/misc/ref/step_3.py.out"
   done

**Comment by sannecottaar** [comment]
I thought I committed a fix to this... Delete the first line of to .out
file.

**Comment by bobmyhill** [comment]
What is the figure meant to look like? I don't see any histograms...

The problem is in the "aspect" variables in the calls to plt.imshow(). Changing aspect_ratio to 'auto' fixes the problem (see #216), although it won't look exactly as it did before.

**Comment by ian-r-rose** [comment]
The figure is meant to look kind of crappy as is. The idea is to optimize for the pv/fp ratio, and then assign some reasonable error bars to the parameters (the ones in there at the moment are way too small). It then generates 1000 realizations of the minerals in order to visualize the effect of the uncertainty.

TL;DR: increase the standard deviations on lines 70 and 71, you should start to see the histograms.

**Comment by bobmyhill** [comment]
Ok, as long as I wasn't meant to see anything! Thanks!
I changed the values, and things appeared as you said. 
PR #216 should be all that's needed to fix the problem.

**Comment by bobmyhill** [comment]
Just seen your PR ... That way is also good!
